parser_tool/static/js/src/case_distribution.js

```diff
@@ -0,0 +1,302 @@
+(function ($) {
+    // const ApiClient = window.app.ApiClient;
+    async function get_radar_config() {
```

Comment:
@eelegiap I think you can declare this function as a normal (non-async) function. There are no asynchronous operations in the function (e.g. HTTP requests). From what I understand, it's creating and configuring an object `RadarChart` that can be used later to draw the chart on the page using d3. So I'd recommend removing the `async` here.